src/Spice86.Core/Emulator/VM/EmulationLoop.cs

```diff
@@ -117,28 +137,34 @@ public class EmulationLoop {
             if (elapsedTimeMilliSeconds > 0) {
                 cyclesPerSeconds = (_cpuState.Cycles * 1000) / elapsedTimeMilliSeconds;
             }
-            _loggerService.Warning("Executed {cycles} instructions in {elapsedTimeMilliSeconds}ms. {cyclesPerSeconds} Instructions per seconds on average over run.", cycles, elapsedTimeMilliSeconds, cyclesPerSeconds);
+            _loggerService.Warning("Executed {Cycles} instructions in {ElapsedTimeMilliSeconds}ms. {CyclesPerSeconds} Instructions per seconds on average over run.", cycles, elapsedTimeMilliSeconds, cyclesPerSeconds);
         }
     }
-    
+
     private bool GenerateUnconditionalGdbBreakpoint() {
-        if (_gdbCommandHandler is null) {
+        if(_gdbCommandHandler == null) {
             return false;
         }
-
         _gdbCommandHandler.Step();
         return true;
     }
-    
-    private void PauseIfAskedTo() {
```

Comment:
I would keep the method so that the if in RunLoop makes sense here

Reply:
Done.